dynamodump/dynamodump.py

```diff
@@ -626,6 +637,11 @@ def do_empty(dynamo, table_name):
     # wait for table creation completion
     wait_for_active_table(dynamo, table_name, "created")
 
+    if streams:
```

Comment:
This PR now seems to contain changes regards to streams which are unrelated to the original change, please remove them

Reply:
fixed